packages/icons/scripts/gradientsToDefs.js

```diff
@@ -0,0 +1,59 @@
+// Mostly stolen from Jake Albaugh https://codepen.io/jakealbaugh/pen/OVrQXY
+
+function gradientsToDefs(source) {
+  // all gradients expression
+  var gradientExp = /(<[a-zA-Z]+Gradient[\S\s]+?<\/[a-zA-Z]+Gradient>)/g;
```

Comment:
💬  That's a bit scary

Reply:
More scariness below 💥

Reply:
Yeah :/ this is in the original PR if you check the reference. I'm trying to integrate this since it was approved and ready to be merged in. However, I was a bit unsure about this functionality, with all the var usage.